packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/datebox.tests.js

```diff
@@ -1189,7 +1189,7 @@ QUnit.module('dateView integration', {
         }, this.instance);
 
         this.popupTitle = function() {
-            return this.popup()._$title.find('.dx-toolbar-label').text();
+            return this.popup()._$topToolbar.find('.dx-toolbar-label').text();
```

Comment:
It would be better to use "public" getters `bottomToolbar/topToolbar` in these places to avoid such refactorings in the future

Reply:
There are no such public properties in the documentation, and there is no guarantee that we won't change this tomorrow: https://js.devexpress.com/jQuery/Documentation/ApiReference/UI_Components/dxPopup/Configuration/.
But I changed it for consistency.

Reply:
Improved here: [f12314916353b33f10d4c7ee4c99a008bcb9d6a4]